README.md

```diff
@@ -1,13 +1,15 @@
 <a href="http://tarantool.org">
 	<img src="https://avatars2.githubusercontent.com/u/2344919?v=2&s=250" align="right">
 </a>
-<!--a href="https://travis-ci.org/tarantool/go-tarantool">
-	<img src="https://travis-ci.org/tarantool/go-tarantool.png?branch=master" align="right">
-</a-->
 
-[![Coverage Status](https://coveralls.io/repos/github/tarantool/go-tarantool/badge.svg?branch=master)](https://coveralls.io/github/tarantool/go-tarantool?branch=master)
+[![Go Reference][godoc-badge]][godoc-url]
+[![Actions Status][actions-badge]][actions-url]
+[![Code Coverage][coverage-badge]][coverage-url]
+[![Telegram][telegram-badge]][telegram-url]
+[![GitHub Discussions][discussions-badge]][discussions-url]
+[![Stack Overflow][stackoverflow-badge]][stackoverflow-url]
 
-# Client in Go for Tarantool 1.6+
+# Client in Go for Tarantool
 
```

Comment:
The package `go-tarantool` contains everything you need to connect to
[Tarantool 1.6+](http://tarantool.org/).

Reply:
Updated.

```diff
--- a/README.md
+++ b/README.md
@@ -11,8 +11,8 @@
 
 # Client in Go for Tarantool
 
-The `go-tarantool` package has everything necessary for interfacing with
-[Tarantool 1.6+](http://tarantool.org/).
+The package `go-tarantool` contains everything you need to connect to
+[Tarantool 1.6+][tarantool-site].
 
 The advantage of integrating Go with Tarantool, which is an application server
 plus a DBMS, is that Go programmers can handle databases and perform on-the-fly
@@ -714,6 +714,7 @@ There are two more connectors from the open-source community available:
 
 See feature comparison in [documentation](https://www.tarantool.io/en/doc/latest/book/connectors/#go-feature-comparison).
 
+[tarantool-site]: https://tarantool.io/
 [godoc-badge]: https://pkg.go.dev/badge/github.com/tarantool/go-tarantool.svg
 [godoc-url]: https://pkg.go.dev/github.com/tarantool/go-tarantool
 [actions-badge]: https://github.com/tarantool/go-tarantool/actions/workflows/testing.yml/badge.svg
```